src/Deployer/Task/After/CachetoolTask.php

```diff
@@ -35,8 +35,6 @@ class CachetoolTask extends TaskBase
 
     public function register(): void
     {
-        after('deploy:symlink', 'cachetool:clear:opcache');
-        after('cachetool:clear:opcache', 'cachetool:cleanup');
```

Comment:
I think the after `after('cachetool:clear:opcache'` can remain, because when you actually make use of that task, you don't need to think about the cleanup.

Otherwise, I agree that this should be optional!